compositor_pipeline/src/pipeline.rs

```diff
@@ -378,6 +378,13 @@ impl Pipeline {
             .outputs
             .get(&output_id)
             .ok_or_else(|| UpdateSceneError::OutputNotRegistered(output_id.clone()))?;
+
+        if let Some(cond) = &output.video_end_condition {
+            if cond.did_output_end() {
+                return Err(UpdateSceneError::UpdateAfterEOS(output_id.clone()));
```

Comment:
What if the audio was already finished, but the video did not? I don't have a strong opinion, but I think I would not throw an error here and at most log a warning here. (the same for audio)

Reply:
Since I also don't have a strong opinion on this, I think I'll ignore all updates after EOS and log a warning instead.